components/prism-tap.js

```diff
@@ -0,0 +1,20 @@
+Prism.languages.tap = {
+	fail: /(    )*not ok[^#\{\n]*/,
+	pass: /(    )*ok[^#\{\n]*/,
+	pragma: /(    )*pragma ([\+-])([a-z]+)/,
```

Comment:
You do not need to escape the `+` in the character class.
Also, the last two pairs of parentheses are not needed.